instagram/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import SearchBar from './components/SearchBar/searchBar';
+import PostContainer from './components/PostContainer/postContainer';
+import dummyData from './dummy-data';
+import './App.css';
+
+class App extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			data: []
+		};
+	}
+
+	componentDidMount() {
+		this.setState({ data: dummyData });
+	}
+
+	filterPosts = str => {
+		const filteredPosts = dummyData.filter(post => post.username.includes(str));
```

Comment:
Nice job making your change to posts outside of your call to `setState()`. Makes the code clean and easy to read/debug 👍